src/utils/AppStateMonitor.ts

```diff
@@ -33,21 +32,16 @@ export default class AppStateMonitor {
         previousState.type === RNAppStateType.Inactive) &&
       nextStateType == RNAppStateType.Active
     ) {
+      // Calculate suspened time in seconds
+      const suspendedTime = (currentTime - previousState.lastUpdated) / 1000
```

Comment:
It would probably be better to leave this as milliseconds. We should probably also try to keep all our "time" state standardized to milliseconds so we don't have to remember what type a duration value is.

Reply:
All units have been converted to milliseconds.